lips/lip-20.md

```diff
@@ -0,0 +1,595 @@
+* [`Libra::fractional_part`](https://github.com/libra/libra/blob/master/language/stdlib/modules/doc/Libra.md#function-fractional_part)
+* [`Libra::currency_code`](https://github.com/libra/libra/blob/master/language/stdlib/modules/doc/Libra.md#function-currency_code)
+* [`Libra::lbr_exchange_rate`](https://github.com/libra/libra/blob/master/language/stdlib/modules/doc/Libra.md#function-lbr_exchange_rate)
+
+### Mutative ("setters")
+1. Update the [`to_lbr_exchange_rate`](#to_lbr_exchange_rate) metadata field for `C` to `new_to_lbr_exchange_rate`.
+  Must be called by an `account` with a [Treasury Compliance role](https://github.com/libra/lip/blob/master/lips/lip-2.md#roles).
+    ```rust
+        Libra::update_lbr_exchange_rate<C>(account: &signer, new_to_lbr_exchange_rate: FixedPoint32)
+    ```
+2. Set the [`can_mint`](#can_mint) metadata field for `C` to the value of `can_mint`.
+  Must be called by an `account` with a [Treasury Compliance role](https://github.com/libra/lip/blob/master/lips/lip-2.md#roles).
+    ```rust
+        Libra::update_minting_ability<C>(account: &signer, can_mint: bool)
+    ```
+
+#### Reference Implementations
+* [`Libra::update_lbr_exchange_rate`](https://github.com/libra/libra/blob/master/language/stdlib/modules/doc/Libra.md#function-update_lbr_exchange_rate)
+* [`Libra::update_minting_ability`](https://github.com/libra/libra/blob/master/language/stdlib/modules/doc/Libra.md#function-update_minting_ability)
+
+### Predicates
+1. Return if the type `C` represents a registered currency on-chain.
+    ```rust
+        Libra::is_currency<C>(): bool
+    ```
+    Related assertion
+    ```rust
+        Libra::assert_is_currency<C>()
+    ```
+2. Return if the type `C` represents a registered SCS currency on-chain.
+    ```rust
+        Libra::is_SCS_currency<C>(): bool
+    ```
+    Related assertion:
+    ```rust
+        Libra::assert_is_SCS_currency<C>()
+    ```
+3. Return if the type `C` represents a registered synthetic currency on-chain.
+    ```rust
+        Libra::is_synthetic_currency<C>(): bool
+    ```
+
+#### Reference Implementations
+
+* [`Libra::is_currency`](https://github.com/libra/libra/blob/master/language/stdlib/modules/doc/Libra.md#function-is_currency)
+    - Related assertion [`Libra::assert_is_currency`](https://github.com/libra/libra/blob/master/language/stdlib/modules/doc/Libra.md#function-assert_is_currency)
+* [`Libra::is_SCS_currency`](https://github.com/libra/libra/blob/master/language/stdlib/modules/doc/Libra.md#function-is_SCS_currency)
+    - Related assertion [`Libra::assert_is_SCS_currency`](https://github.com/libra/libra/blob/master/language/stdlib/modules/doc/Libra.md#function-assert_is_SCS_currency)
+* [`Libra::is_synthetic_currency`](https://github.com/libra/libra/blob/master/language/stdlib/modules/doc/Libra.md#function-is_synthetic_currency)
+
+---
+# <a name="currency_events">Currency-Related Events</a>
```

Comment:
If it's not too much work, could be useful to discuss reconciliation of events here. E.g., 

sum(values in all mint events for C) - sum(values in all burn events for C) = total value for C